context/run_commands.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
+# give sudo permission for rapids user to run apt/yum (user creation is postponed
+# to the entrypoint, so we can create a user with the same id as the host)
+if cat /etc/os-release | grep centos > /dev/null; then
+  PKG_MGR="yum";
+else
+  PKG_MGR="apt";
+fi
+echo "rapids ALL=NOPASSWD: /usr/bin/${PKG_MGR}" >> /etc/sudoers
```

Comment:
Any reason why its limited to just the package manager instead of `ALL`?

Reply:
nope, no reason in particular. this was copied from the set of files that Mike asked me to reference, but I'm fine with changing it if you think it needs to be adjusted

https://github.com/mike-wendt/docker-images/blob/master/scripts/run_commands#L31